src/main/java/io/gingersnapproject/k8s/configuration/DatabaseServiceBindingConverter.java

```diff
@@ -52,6 +52,9 @@ public Optional<ServiceBindingConfigSource> convert(List<ServiceBinding> allBind
             .stream()
             .collect(Collectors.toMap(e -> Database.property(e.getKey()), Map.Entry::getValue));
       properties.put(Database.property("type"), type);
+      if (log.isDebugEnabled()) {
+         properties.forEach((k, v) -> log.debugf("%s=%s", k, v));
```

Comment:
Some sort of sensitive information here?

Reply:
Yes the username and password will be output, however we'll never release the Operator with DEBUG logs enabled. So I don't think this is a big concern. Having these logs available for debugging locally is invaluable though